samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -36,16 +37,24 @@
 #define bytes2KiB(Bytes)	(Bytes / (1024u))
 #define bytes2MiB(Bytes)	(Bytes / (1024u * 1024u))
 
+#if defined(CONFIG_SAMPLE_BIG_HTTP_DL_MAX_URL_LENGTH)
+#define MAX_URL_LENGTH CONFIG_SAMPLE_BIG_HTTP_DL_MAX_URL_LENGTH
+#else
+#define MAX_URL_LENGTH 256
+#endif
+
+#if defined(CONFIG_SAMPLE_BIG_HTTP_DL_NUM_ITER)
+#define NUM_ITER CONFIG_SAMPLE_BIG_HTTP_DL_NUM_ITER
+#else
+#define NUM_ITER 0
+#endif
+
```

Comment:
Both configs are provided via Kconfig for the sample, shouldn't then the "defaults" be set there?

Reply:
Understood.